packages/google_sign_in/google_sign_in_ios/lib/src/messages.g.dart

```diff
@@ -0,0 +1,376 @@
+      writeValue(buffer, value.encode());
+    } else {
+      super.writeValue(buffer, value);
+    }
+  }
+
+  @override
+  Object? readValueOfType(int type, ReadBuffer buffer) {
+    switch (type) {
+      case 128:
+        return InitParams.decode(readValue(buffer)!);
+      case 129:
+        return TokenData.decode(readValue(buffer)!);
+      case 130:
+        return UserData.decode(readValue(buffer)!);
+      default:
+        return super.readValueOfType(type, buffer);
+    }
+  }
+}
+
+class GoogleSignInApi {
+  /// Constructor for [GoogleSignInApi].  The [binaryMessenger] named argument is
+  /// available for dependency injection.  If it is left null, the default
+  /// BinaryMessenger will be used which routes to the host platform.
+  GoogleSignInApi({BinaryMessenger? binaryMessenger})
+      : _binaryMessenger = binaryMessenger;
+  final BinaryMessenger? _binaryMessenger;
+
+  static const MessageCodec<Object?> codec = _GoogleSignInApiCodec();
+
+  /// Initializes a sign in request with the given parameters.
+  Future<void> init(InitParams arg_params) async {
+    final BasicMessageChannel<Object?> channel = BasicMessageChannel<Object?>(
+        'dev.flutter.pigeon.google_sign_in_ios.GoogleSignInApi.init', codec,
+        binaryMessenger: _binaryMessenger);
+    final List<Object?>? replyList =
+        await channel.send(<Object?>[arg_params]) as List<Object?>?;
+    if (replyList == null) {
+      throw PlatformException(
+        code: 'channel-error',
+        message: 'Unable to establish connection on channel.',
+      );
+    } else if (replyList.length > 1) {
+      throw PlatformException(
+        code: replyList[0]! as String,
+        message: replyList[1] as String?,
+        details: replyList[2],
+      );
+    } else {
+      return;
+    }
+  }
+
+  /// Starts a silent sign in.
+  Future<UserData> signInSilently() async {
+    final BasicMessageChannel<Object?> channel = BasicMessageChannel<Object?>(
+        'dev.flutter.pigeon.google_sign_in_ios.GoogleSignInApi.signInSilently',
+        codec,
+        binaryMessenger: _binaryMessenger);
+    final List<Object?>? replyList = await channel.send(null) as List<Object?>?;
+    if (replyList == null) {
+      throw PlatformException(
+        code: 'channel-error',
+        message: 'Unable to establish connection on channel.',
+      );
+    } else if (replyList.length > 1) {
+      throw PlatformException(
+        code: replyList[0]! as String,
+        message: replyList[1] as String?,
+        details: replyList[2],
+      );
+    } else if (replyList[0] == null) {
+      throw PlatformException(
+        code: 'null-error',
+        message: 'Host platform returned null value for non-null return value.',
+      );
+    } else {
+      return (replyList[0] as UserData?)!;
+    }
+  }
+
+  /// Starts a sign in with user interaction.
+  Future<UserData> signIn() async {
+    final BasicMessageChannel<Object?> channel = BasicMessageChannel<Object?>(
```

Comment:
reported here, but this line fails when re-try signin after 403 error

https://github.com/flutter/flutter/issues/136541